.gitlab-ci.yml

```diff
@@ -5,47 +5,45 @@
 # pipelines can be triggered manually in the web
 # setting DEPLOY_TAG will only deploy the tagged image
 
-
 stages:
-    - build
+  - build
 
 variables:
-  GIT_STRATEGY:                    fetch
-  GIT_SUBMODULE_STRATEGY:          recursive
-  GIT_DEPTH:                       3
-  CARGO_INCREMENTAL:               0
-  CARGO_HOME:                      "/ci-cache/${CI_PROJECT_NAME}/cargo/${CI_JOB_NAME}"
-  REGISTRY:                        registry.parity.io/parity/infrastructure/scripts
-
+  GIT_STRATEGY: fetch
+  GIT_SUBMODULE_STRATEGY: recursive
+  GIT_DEPTH: 3
+  CARGO_INCREMENTAL: 0
+  CARGO_HOME: "/ci-cache/${CI_PROJECT_NAME}/cargo/${CI_JOB_NAME}"
+  REGISTRY: registry.parity.io/parity/infrastructure/scripts
 
-.docker-env:                       &docker-env
-  image:                           ${REGISTRY}/contracts-ci-linux:latest
+.docker-env: &docker-env
+  image: ${REGISTRY}/contracts-ci-linux:latest
   before_script:
     - rustup show
     - cargo --version
     - sccache -s
   only:
     - master
-    - /^v[0-9]+\.[0-9]+.*$/        # i.e. v1.0, v2.1rc1
+    - /^v[0-9]+\.[0-9]+.*$/ # i.e. v1.0, v2.1rc1
     - schedules
     - web
-    - /^[0-9]+$/                   # PRs
+    - /^[0-9]+$/ # PRs
   retry:
     max: 2
     when:
       - runner_system_failure
       - unknown_failure
       - api_failure
-  dependencies:                    []
-  interruptible:                   true
+  dependencies: []
+  interruptible: true
   tags:
     - linux-docker
 
 #### stage:                        build
 
 build-and-test:
-  stage:                           build
-  <<:                              *docker-env
```

Comment:
can you please return all changed margings back? It's how this config looks in all the projects...